scm/etc/scripts/MOSAiC_AMPS_forcing_file_generator.py

```diff
@@ -0,0 +1,590 @@
+# spr.contour_plot_firl(time, levels, v_advec_T, np.min(v_advec_T), np.max(v_advec_T), 'v advec T', 'time', 'pressure', 'v_advec_T.eps', y_inverted = True)
+# spr.contour_plot_firl(time, levels, dT_dt, np.min(dT_dt), np.max(dT_dt), 'total T tend', 'time', 'pressure', 'dT_dt.eps', y_inverted = True)
+
+#h_advec_qt = np.zeros((levels.size,time.size),dtype=float)
+h_advec_qt = nc_fid.variables['h_advec_qt'][:] #kg/kg/s
+h_advec_qt = np.swapaxes(h_advec_qt, 0, 1) #swap the time and levels axis
+h_advec_qt = h_advec_qt/(1.0 + qt_mr)**2 #convert to specific humidity
+
+h_advec_qi = np.zeros((levels.size,time.size),dtype=float)
+h_advec_ql = np.zeros((levels.size,time.size),dtype=float)
+
+#v_advec_qt = np.zeros((levels.size,time.size),dtype=float)
+v_advec_qt = nc_fid.variables['v_advec_qt'][:] #kg/kg/s
+v_advec_qt = np.swapaxes(v_advec_qt, 0, 1) #swap the time and levels axis
+v_advec_qt = v_advec_qt/(1.0 + qt_mr)**2 #convert to specific humidity
+
+dq_dt = np.zeros((levels.size,time.size),dtype=float)
+#dq_dt = nc_fid.variables['dqdt'][:]
+#dq_dt = np.swapaxes(dq_dt, 0, 1)*1.0E-3/3600.0 #swap the time and levels axis, convert to K/s
+
+#h_advec_qt = h_advec_qt*86400.0
+#v_advec_qt = v_advec_qt*86400.0
+#dq_dt = dq_dt*86400.0
+#spr.contour_plot_firl(time, levels, h_advec_qt, np.min(h_advec_qt), np.max(h_advec_qt), 'h advec q', 'time', 'pressure', 'h_advec_q.eps', y_inverted = True)
+#spr.contour_plot_firl(time, levels, v_advec_qt, np.min(v_advec_qt), np.max(v_advec_qt), 'v advec q', 'time', 'pressure', 'v_advec_q.eps', y_inverted = True)
+#spr.contour_plot_firl(time, levels, dq_dt, np.min(dq_dt), np.max(dq_dt), 'total q tend', 'time', 'pressure', 'dq_dt.eps', y_inverted = True)
+
+#phi_sfc = nc_fid.variables['phis'][:]
+#z_sfc = nc_fid.variables['alt'][:]
+height = nc_fid.variables['height'][:]
+#height = ffc.get_height_from_pres(T_abs[:,0],levels,z_sfc)
+
+#the following variables are not in this forcing file, but are included in other cases
+soil_depth = np.zeros(4,dtype=float)
+soil_depth[:] = [0.1, 0.4, 1.0, 2.0]
+
+#rad_heating = nc_fid.variables['dT_dt_rad'][:] #K/s
+#rad_heating = np.swapaxes(rad_heating, 0, 1) #swap the time and levels axis
+rad_heating = np.zeros((levels.size,time.size),dtype=float)
+u_g = np.zeros((levels.size,time.size),dtype=float)
+v_g = np.zeros((levels.size,time.size),dtype=float)
+
+ozone = np.zeros((levels.size,time.size),dtype=float)
+ozone = ozone + 2.e-8
+
+tiice = np.zeros((ice_thickness.size),dtype=float)
+##tiice[0] = T_surf[start_t_index] 
+tiice[0] = 271.35 + .75*(T_surf[start_t_index] - 271.35)
+tiice[1] = 271.35 + .25*(T_surf[start_t_index] - 271.35)
+stc      = tiice[0]
+smc      = 0.33
+slc      = 0.33
+hice     = 0.3
+slmsk    = 2.0    #sea ice
+tsfco    = T_surf[start_t_index]
+weasd    = 200.0 #mm water equivalent snow depth
+fice     = 1.0
+tisfc    = tsfco
+snwdph   = 2.e-4
+tg3      = 271.35
+zorl     = 15.0
+alvsf    = 0.06
```

Comment:
Updated these (alvsf, alnsf, alvwf, alnwf, facsf, facwf). The previous values were from the GABLS3 case, but those are specific to that location. These are the values from the UFS fix files for this lat/lon. It doesn't seem to change the results though.